data.json

```diff
@@ -2415,6 +2452,114 @@
     "type": "confs",
     "comments": [ "Somewhat Vim-like keys are available, but shortcuts do conflict with Vim-like browsers/plugins, which makes them significantly less useful." ],
     "ui": "web"
+},
+{
+    "name": "vim-anywhere",
+    "id": "vim-anywhere",
+    "descr": "Spawn a vim buffer from any text input in the operating system.",
+    "url": "https://github.com/cknadler/vim-anywhere",
+    "license": "MIT",
+    "languages": [ "Bash" ],
+    "category": "Other",
+    "type": "apps",
+    "comments": [ "" ],
+    "ui": "terminal"
+},
+{
+    "name": "ipython",
+    "id": "ipython",
+    "descr": "Interactive Python shell.",
+    "url": "https://ipython.org/",
+    "license": "BSD-like",
+    "languages": [ "python" ],
+    "category": "Shells",
+    "type": "confs",
+    "comments": [ "It no longer uses readline for input, so it must be configured in the ipython profile `~/.ipython/profile_default/ipython_config.py` itself ([reference](https://gist.github.com/sstirlin/c3c207b1052b613ab9554b4ebdfc3f35)):\n```ini\nc.TerminalInteractiveShell.editing_mode = \"vi\"\n```" ],
+    "ui": "CLI"
+},
+{
+    "name": "GHCi",
+    "id": "",
+    "descr": "Interactive Haskell environment.",
+    "url": "https://wiki.haskell.org/GHC/GHCi",
+    "license": "BSD-like",
+    "languages": [ "haskell" ],
+    "category": "shells",
+    "type": "confs",
+    "comments": [ "It uses [haskeline](https://hackage.haskell.org/package/haskeline) instead of readline. Put this in `~/haskeline`:\n```yaml\neditMode: Vi\n```" ],
+    "ui": "cli"
+},
+{
+    "name": "vimpdb",
+    "id": "",
+    "descr": "An integration of the python debugger `pdb` and vim",
+    "url": "https://github.com/gotcha/vimpdb",
+    "license": "MIT",
+    "languages": [ "Python" ],
+    "category": "Debugger",
+    "type": "app",
+    "comments": [ "" ],
```

Comment:
Here and in several other new entries: if there are not comments, no need to add an empty one.

Reply:
Makes sense!